R/bert_classification-fit.R

```diff
@@ -31,20 +37,19 @@
 #'   * A __matrix__ with 1 factor column.
 #'   * A factor __vector__.
 #'
-#' @param valid_x Depending on the context:
-#'   * A number between 0 and 1, representing the fraction of data to use for
-#'     model validation.
-#'   * Predictors in the same format as `x`. These predictors will be used for
-#'     model validation.
-#'   * `NULL`, in which case no data will be used for model validation.
+#' @param valid_x Depending on the context: * A number between 0 and 1,
```

Comment:
whoops, somehow this got reformatted in this commit.

Reply:
Grr. Comment reflow doesn't understand markdown.